### ISSUE TITLE
Adjust sentence length

### ISSUE THREAD

**Issue by gerardosabetta** [Vaibhavs10/insanely-fast-whisper#130]
Hi, I'm trying to get fast paced subtitles showing few words at a time. I was wondering if there is a way to tweak this.

Thanks in advance

**Comment by gerardosabetta** [comment]
> Hi @gerardosabetta - do you mean you'd like to get streaming output? we do not support that at the moment.

Hi @Vaibhavs10 thanks for the reply. No that's not what I meant sorry. Let me rephrase it and give a concrete example with subtitles from one of the notebooks in this repo.

This segment is almost 4 seconds long and contains a full sentence
```
[0.00s -> 3.92s]  We have been a misunderstood and badly mocked org for a long time.
```

I'd like to customize the segment length and maybe have something like this:
```
[0.00s -> 0.42s] We have been
[0.42s -> 1.92s] a misunderstood and badly 
[1.92s -> 3.92s] mocked org for a long time.
```

Reason being that I want to subtitle vertical videos and I few words at a time work better with the limited real estate there is on a phone screen

**Comment by MaximeDde** [comment]
I second this, the usage could be very useful. For now, at least, it's done outside...

**Comment by heyimjonas** [comment]
You could look into other projects like [whisper-timestamped](https://github.com/linto-ai/whisper-timestamped )for accurate timestamps for words.
